src/OpenTelemetry.Extensions/Internal/OpenTelemetryExtensionsEventSource.cs

```diff
@@ -44,4 +44,10 @@ public void LogRecordFilterException(string? categoryName, string exception)
     {
         this.WriteEvent(2, categoryName, exception);
     }
+
+    [Event(3, Message = "Baggage key predicate function threw exeption, no baggage entries will not be added to the activity. Exception: '{0}'", Level = EventLevel.Warning)]
```

Comment:
something is wrong with this error message..
It is not clear if all baggage entries will be dropped or a particular baggageentry will be dropped ?

Reply:
Updated message with baggage key in [87478e0](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1816/commits/87478e0c629af630e1aaf7a18ad0e497ebcf3f7a)